src/components/App.vue

```diff
@@ -2,7 +2,7 @@
     <div id="app">
         <div class="https-warning" v-if="isInsecure">
             <strong class="https-warning__headline">{{ 'ui.app.httpsHeadline' | translate }}</strong>
-            <span class="https-warning__description">{{ 'ui.app.httpsDescription' | translate }}</span>
+            <span class="https-warning__description"> {{ 'ui.app.httpsDescription' | translate }} </span>
```

Comment:
Why did you add these whitespaces?

Reply:
Because they're missing there:

![cm](https://user-images.githubusercontent.com/1218766/49442492-f2df2e80-f7c9-11e8-82b0-ba5db93f4506.png)

Of course, you could also add these spaces somewhere else.

Reply:
There is a newline, which should be converted to spaces in HTML? 🤔